### PR TITLE
Less Ass README closes #476

### PR DESCRIPTION
**Pull request by mikeynap**

### CODE REVIEW

**Comment by mikeynap** [comment]
Closes #476

File: docs/current/README.md [hunk 1]
```diff
@@ -155,8 +162,10 @@ Log:
...
 | All    | SavePath              | no                | Base folder for vaquero save files                                | /var/vaquero       |
-| Agent  | AgentAPI/InsecureAddr | no                | IP Address on which to serve the agent REST API                   | 127.0.0.1          |
-| Agent  | AgentAPI/InsecurePort | no                | Port on which to serve the agent REST API                         | 24602              |
+| Agent  | ServerClient/Addr     | no                | IP Address of vaquero-server's API.                               | 127.0.0.1          |
+| Agent  | ServerClient/Port     | no                | Port of vaquero-server's API                                      | 24601              |
+| Agent  | ServerClient/         | no                | vaquero-agent should allow a self-signed server certificate.      | false
+|          InsecureSkipVerify    |                   |                                                                   |                    |
 | Agent  | Assets/CdnScheme      | no                | Cdn scheme                                                        | none               |
```
Comment (stephenrlouie) [inline docs/current/README.md]: `http`, not none

File: docs/current/README.md [hunk 2]
```diff
@@ -155,8 +162,10 @@ Log:
...
 | Agent  | Assets/CdnAddr        | no                | The address of the cdn endpoint to reverse proxy to               | http               |
```
Comment (stephenrlouie) [inline docs/current/README.md]: none

File: docs/current/README.md [hunk 3]
```diff
@@ -155,8 +162,10 @@ Log:
...
 | Agent  | Assets/CdnPort        | no                | The port of the cdn endpoint to reverse proxy to                  | 0                  |
```
Comment (stephenrlouie) [inline docs/current/README.md]: none

File: docs/current/README.md [hunk 4]
```diff
@@ -165,8 +174,10 @@ Log:
 | Agent  | AssetServer/Scheme    | no                | Asset server scheme : http / https                                | http               |
 | Agent  | AssetServer/BaseDir   | no                | Agent directory to serve files from                               | /var/vaquero/files |
 | Agent  | DHCPMode              | no                | Agent DHCP Mode: server / proxy                                   | server             |
-| Server | ServerAPI/Address     | no                | The IP Address to serve the server REST API on                    | 127.0.0.1          |
+| Server | ServerAPI/Addr        | no                | The IP Address to serve the server REST API on                    | 127.0.0.1          |
 | Server | ServerAPI/Port        | no                | The port to serve the server REST API on                          | 24601              |
+| Server | ServerAPI/PrivateKey  | yes               | vaquero-server HTTPS/TLS Private Key                              |                    |
```
Comment (stephenrlouie) [inline docs/current/README.md]: can you put `none` in the default column

File: docs/current/README.md [hunk 5]
```diff
@@ -155,18 +162,22 @@ Log:
...
-| Agent  | Assets/CdnAddr        | no                | The address of the cdn endpoint to reverse proxy to               | http               |
+| Agent  | ServerClient/Addr     | no                | IP Address of vaquero-server's API.                               | 127.0.0.1          |
+| Agent  | ServerClient/Port     | no                | Port of vaquero-server's API                                      | 24601              |
+| Agent  | ServerClient/         | no                | vaquero-agent should allow a self-signed server certificate.      | false
+|          InsecureSkipVerify    |                   |                                                                   |                    |
+| Agent  | Assets/CdnScheme      | no                | Cdn scheme                                                        | http               |
+| Agent  | Assets/CdnAddr        | no                | The address of the cdn endpoint to reverse proxy to               | none               |
 | Agent  | Assets/CdnPort        | no                | The port of the cdn endpoint to reverse proxy to                  | 0                  |
```
Comment (stephenrlouie) [inline docs/current/README.md]: none

File: docs/current/README.md [hunk 6]
```diff
@@ -155,18 +162,22 @@ Log:
...
-| Server | ServerAPI/Address     | no                | The IP Address to serve the server REST API on                    | 127.0.0.1          |
+| Server | ServerAPI/Addr        | no                | The IP Address to serve the server REST API on                    | 127.0.0.1          |
 | Server | ServerAPI/Port        | no                | The port to serve the server REST API on                          | 24601              |
+| Server | ServerAPI/PrivateKey  | yes               | vaquero-server HTTPS/TLS Private Key                              | none               |
+| Server | ServerAPI/PublicKey   | yes               | vaquero-server HTTPS/TLS Public Key                               | none               |
 | Server | Etcd/Endpoints        | no                | etcd initial cluster endpoints: format- e1,e2,e3                  | 127.0.0.1:2379     |
 | Server | Etcd/Retry            | no                | number of retries for etcd operations                             | 3                  |
 | Server | Etcd/Timeout          | no                | etcd dial and request timeout, in seconds                         | 5                  |
```
Comment (stephenrlouie) [inline docs/current/README.md]: timeout is 2 right now

**Pull request merged**